README.rst

```diff
@@ -1,3 +1,9 @@
+Important notice when upgrading from ``[TODO: INSERT FINAL 1DB RELEASE HERE]`` or earlier
+-----------------------------------------------------------------------------------------
+
+Up to and including release ``[TODO: INSERT FINAL 1DB RELEASE HERE]``, this project (KoBoCAT) and `KPI <https://github.com/kobotoolbox/kpi>`_ both shared a common Postgres database. They now each have their own. **If you are upgrading an existing single-database installation, you must follow** `these instructions <TODO: LINK>`_ to migrate the (smaller) KPI tables to a new database and adjust your configuration appropriately.
```

Comment:
#TODO Link missing

Reply:
Quick question: will the ToDo link get updated/completed in the semi near-future or is there any general timeline for that?

Reply:
@tolexy, it is updated now